### DISCUSSION TITLE
Should `Injected` be shown if no connectors are discovered via `mipd`?

### DISCUSSION THREAD

**Discussion by frankfka** [wevm/wagmi#3527] [Ideas]
Hello!

In the `2.x` versions, `mipd` makes supporting multiple injected wallets super-easy. However, this does not work for _all_ cases. For example, the Trust Wallet mobile browser does not seem to be picked up by the `mipd` library.

In cases such as these, should `wagmi` show a generic `injected` option when calling `useConnectors`?

**Comment by tmm** [comment]
It works for all EIP-6963 cases, which is what the feature is for. You should include `injected()` in your connectors config if you want to handle providers that don't support EIP-6963.
